### ISSUE TITLE
FEAT: Ability to group and sort the connections

### ISSUE THREAD

**Issue by Ancinek** [beekeeper-studio/beekeeper-studio#440]
**Is your feature request related to a problem? Please describe.**
First thing – I love the product. I was searching for something like this – a modern, simple db handler – and I found it!

When working on different projects, I'm dealing with multiple environments. One thing I liked in PGAdmin is that I could create a group inside the connections (like e.g. `Project A`) and then add multiple connections to it (e.g. `Production`, `Staging`, `Local`). It would be great to have a feature like this here as well!

The other thing is connections sorting – a feature that would make managing the connections much easier.

**Describe the solution you'd like**
1. Ability to add groups
2. Ability to drag/drop the connections/connection groups

**Describe alternatives you've considered**
When it comes to sorting – I would be fine with an option in the connection menu saying e.g. `Move up` and `Move down`

**Additional context**
Here is an example of the UI in PGAdmin 4
<img width="497" alt="Screenshot 2020-11-10 at 17 34 19" src="https://user-images.githubusercontent.com/16444141/98702875-3742b400-237b-11eb-8cc9-dddd3a97da82.png">

**Comment by rathboma** [comment]
Yes! We are doing this. Slow going right now with the pandemic but we'll get there!

**Comment by Ancinek** [comment]
@rathboma That's brilliant! Thanks for your work!

**Comment by evs-xsarus** [comment]
https://github.com/beekeeper-studio/beekeeper-studio/issues/372

**Comment by not-night-but** [comment]
This is solved with connection folders which are coming out in the next release
